packages/fxa-shared/nestjs/sentry/reporting.ts

```diff
@@ -46,6 +49,22 @@ export function isApolloError(err: Error): boolean {
   return false;
 }
 
+export function isOriginallyHttpError(
```

Comment:
This check used to be:

```
          // Skip errors with a status already set or already reported
          if ((err.originalError as any)?.status) {
            continue;
          }
```

Was the difference that we needed `>= 400`? Should we leave a comment similarly about "with a status already set or already reported"?

Reply:
I don't know how it would happen but a 3XX or 2XX response is definitely not an error, and shouldn't be hitting this code. If it does, something is really off. I debated just checking if the status of the object existed... I can also see this giving a false positive if some other 'error' object just so happened to have to have a status field... Probably over thinking it.